### PR TITLE
add django-imgix

### PR DESCRIPTION
**Pull request by frederickfogerty**
This PR adds django-imgix to this list of awesome components ✨ This is an awesome library because it makes images rendered by it automatically more performant, and responsive! The repository has been around for years, is well-documented, and commercially-supported. It is a commercial library, but it has a very generous free tier which is enough for most people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ _For a complete listing of all available packages, see [Django Packages](https:/
 
 ### Logging
 - [django-guid](https://github.com/JonasKs/django-guid) - Inject a GUID (Correlation-ID) into every log message in a Django request.
+- [django-imgix](https://github.com/imgix/django-imgix) - Transform, optimize, and intelligently cache your entire image library for fast websites and app. Commercial project w/ generous free tier.
```
Comment (JonasKs) [inline README.md:120]: Think you misplaced it? Don't really have anything to do with logging?

Comment (frederickfogerty) [reply README.md:120]: Ah, whoops! Don't know how it got there. Thanks for pointing this out, I'll fix it now @JonasKs.

**Comment by frederickfogerty** [comment]
@JonasKs updated!

**Comment by jefftriplett** [comment]
hi @frederickfogerty, thank you for the contribution. @wsvincent and I will discuss it because we haven't quite figured out how we want to handle sponsored libraries or libraries which wrap commercial products yet. 

So far, we support libraries that have OSS services, whereas while imgix has a free tier (which I am a fan of), and we want to make sure being included in the list isn't confused with a product endorsement. 

We'll talk about it and follow back up if we come up with a policy that addresses our concerns.

**Comment by frederickfogerty** [comment]
Great, thanks @jefftriplett 🙌

**Pull request closed without merging**